shader/antialiasing/cmaa2/edge_color2x2.comp

```diff
@@ -0,0 +1,256 @@
+#version 460
```

Comment:
Shader code feel like HLSL in many ways...

Is it from Intel's demo, or is it rewrite of original DX code? If rewrite, need to adjust style and language to glsl:
* `g_` opengothic doesn't use it
* function name state with lower-case
* avoid bindings in header file (if unavoidable - put all of them into single header, such as cmaa2_common.glsl)

Reply:
I put all common (usages in more than one pass) bindings in `cmaa2_common.glsl`